lib/sdf-server/src/server/service/v2/variant.rs

```diff
@@ -72,6 +128,11 @@ pub async fn list_schema_variants(
         }
     }
 
+    if let Err(err) = determine_can_update_many(&ctx, &mut schema_variants, &raw_access_token).await
+    {
+        error!(?err, "could not perform 'determine_can_update_many'");
+    }
```

Comment:
Wrapped in an infallible wrapper for now because determining `canUpdate` should not error out the route... yet. I suspect this logic moves into its own route in the near future anyway, so let's not cause any problems.